### PR TITLE
Release notes for Edge-19.5.1

### PR DESCRIPTION
**Pull request by adleong**
* CLI
  * Added a `linkerd check config` command for verifying that
    `linkerd install config` was successful
  * Improved the help documentation of `linkerd install` to clarify flag usage
  * Added support for private Kubernetes clusters by changing the CLI to connect
    to the control plane using a port-forward (thanks, @jackprice!)
* Controller
  * Fixed pod creation failure when a `ResourceQuota` exists by adding a default
    resource spec for the proxy-init init container
* Proxy
  * Replaced the fixed reconnect backoff with an exponential one (thanks,
    @zaharidichev!)
  * Fixed an issue where load balancers can become stuck
* Internal
  * Fixed integration tests by adding known proxy-injector log warning to tests

Signed-off-by: Alex Leong <alex@buoyant.io>

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+  * Added a `linkerd check config` command for verifying that
+    `linkerd install config` was successful
+  * Improved the help documentation of `linkerd install` to clarify flag usage
+  * Added support for private Kubernetes clusters by changing the CLI to connect
+    to the control plane using a port-forward (thanks, @jackprice!)
+* Controller
+  * Fixed pod creation failure when a `ResourceQuota` exists by adding a default
+    resource spec for the proxy-init init container
+* Proxy
+  * Replaced fixed reconnect backoff with exponential one (thanks,
```
Comment (klingerf) [inline CHANGES.md]: At first I thought "fixed" was a verb but now I realize it's an adjective. Maybe some articles would help. Can you rephrase as "Replace _the_ fixed reconnect backoff with _an_ exponential one"?

File: CHANGES.md [hunk 2]
```diff
@@ -1,3 +1,23 @@
...
+    @zaharidichev!)
+  * Support endpoint weights
```
Comment (klingerf) [inline CHANGES.md]: How about "Added support for endpoint weights" for consistency with the tense of the other bullet points?

Comment (wmorgan) [inline CHANGES.md]: Can we get a little more clarity on what this means and how one can use it?

File: CHANGES.md [hunk 3]
```diff
@@ -1,3 +1,23 @@
...
+  * Fixed an issue where load balancers can become stuck by removing buffers
```
Comment (wmorgan) [inline CHANGES.md]: IMO remove "removing buffers from endpoint stacks" since that has little meaning to anyone outside the implementation

File: CHANGES.md (hunk 2 above)
Comment (adleong) [reply CHANGES.md]: Whoops, meant to remove this since it's not a usable end-to-end feature yet.
